docs/installation/sources/sources_windows.rst

```diff
@@ -124,6 +124,10 @@ Dependencies
 * :ref:`asiotinyxml2_sw`
 * :ref:`openssl_sw`
 
+.. seealso::
+  For further information about this Fast DDS version dependencies and other products related versions, please check
+  the Fast DDS :ref:`related_versions` section.
```

Comment:
If the suggestion  about the name of the `related_versions` section is applied, please change also here. Moreover, if we change the name we can refer directly to the specific section inside the page, i.e. `Build time dependencies`